staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -340,4 +347,6 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	ConsistentListFromCache: {Default: false, PreRelease: featuregate.Alpha},
 
 	ZeroLimitedNominalConcurrencyShares: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.32
+
+	SeparateCacheWatchRPC: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
nit: put this between `RetryGenerateName` and `ServerSideApply`

Reply:
Done